src/nevermined/Nfts.ts

```diff
@@ -631,6 +632,7 @@ export class Nfts extends Instantiable {
             nftAmount,
             currentNftHolder,
             consumer,
+            [86400, 86400, 86400],
```

Comment:
do we want to set this value as default? I think it would be better to expose this in the method arguments.
Also why this value?

Reply:
1 day, for starters...

Reply:
I also had this assumption but I wanted to get this one going, and then create a separate issue of doing exactly what you said once there is a UI to test it out with.

Reply:
Still you should be able to pass that instead of harcoding the values. I'm sure you will want to change them for testing purposes.

Also I'm not sure if timelocks use actual seconds. Don't they use block number?

We could also update the secondary market integration tests to add this case where a buyer wants to cancel the payment. I think it would make it easier here in the tests to identify potential problems instead of autonomies